src/pool/srv_pool.c

```diff
@@ -2040,14 +2104,35 @@ read_db_for_stepping_up(struct pool_svc *svc, struct pool_buf **map_buf_out,
 			DP_UUID(svc->ps_uuid));
 	}
 
+	prop_entry = daos_prop_entry_get(prop, DAOS_PROP_PO_GLOBAL_VERSION);
+	D_ASSERT(prop_entry != NULL);
+	arg.global_ver = prop_entry->dpe_val;
```

Comment:
By the way, the pool global version is already loaded into `svc->ps_global_version`. Just FYI; getting from `prop` is okay too.

Reply:
ah, ok